tests/foreman/smoke/test_ui_smoke.py

```diff
@@ -46,7 +46,7 @@ class SmokeTestCase(UITestCase):
     """End-to-end tests using the ``WebUI``."""
 
     def test_positive_find_default_org(self):
-        """@Test: Check if :data:`robottelo.constants.DEFAULT_ORG` is present
```

Comment:
I spoke to @elyezer and I will revert these specific doc related changes.  These will be addressed after testimony updates.